render-wasm/src/render.rs

```diff
@@ -113,11 +109,10 @@ impl RenderState {
             skia::SamplingOptions::new(skia::FilterMode::Linear, skia::MipmapMode::Nearest);
         let surfaces = Surfaces::new(&mut gpu_state, (width, height), sampling_options);
 
-        let fonts = FontStore::new();
+        let fonts: FontStore = FontStore::new();
```

Comment:
do you need to have the type explicitely setted?

Reply:
not needed, let me remove it! 👍🏽